manifests/w/WorksMobile/NAVERWORKS/4.0.1.1/WorksMobile.NAVERWORKS.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Automatically updated by the winget bot at 2024/May/28
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: WorksMobile.NAVERWORKS
+PackageVersion: 4.0.1.1
+PackageLocale: en-US
+Publisher: Works Mobile Corp.
+PublisherUrl: https://naver.worksmobile.com
+PublisherSupportUrl: https://guide.worksmobile.com/kr/
+PrivacyUrl: https://www.ncloud.com/policy/infou/infou
+Author: Works Mobile Corp.
+PackageName: NAVER WORKS

```

```suggestion
PackageName: WORKS
```
Should match control panel